src/routes/_authenticated/lists.tsx

```
@@ -0,0 +1,65 @@
+          <AddList
+            onSuccess={refetchWishlists}
+          />
+        </div>
+      </div>
+      <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4">
+        {isLoading && <div>Loading...</div>}
+        {!isLoading && wishlists?.map((list) => (
+          <WishlistCard
+            list={list}
+            onClick={() => { }}
+          />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)